domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -203,12 +239,14 @@ constructor(
                 .tryApplyAspectRatioForExternalCapture(externalImageCapture)
                 .tryApplyImageFormatConstraints()
 
-        imageCaptureUseCase = ImageCapture.Builder()
+        val imageCaptureBuilder = ImageCapture.Builder()
             .setResolutionSelector(
                 getResolutionSelector(
                     settingsRepository.defaultCameraAppSettings.first().aspectRatio
                 )
-            ).build()
+            )
+        onCaptureCompletedCallback(imageCaptureBuilder)
```

Comment:
We'll probably need to have capture completed callbacks for Preview and ImageCapture Usecases. So maybe it's best to inline the callbacks where we setup the builders.

For this specific benchmark, I think we need to measure the Preview usecase.